src/vs/workbench/electron-browser/main.contribution.ts

```diff
@@ -226,6 +226,24 @@ configurationRegistry.registerConfiguration({
 			'description': nls.localize('closeOnFocusLost', "Controls if Quick Open should close automatically once it loses focus."),
 			'default': true
 		},
+		'workbench.quickOpen.prefillFromSelection': {
+			'type': 'boolean',
+			'default': true,
+			'description': nls.localize('quickopen.prefillFromSelection', "Controls if selected text should prefill quickopen query"),
+		},
+		'workbench.quickOpen.prefillPrefixes': {
+			'type': 'array',
+			'items': {
+				'type': ['string', 'null']
+			},
+			'default': [
+				null,
```

Comment:
Not obvious for what null stands for.

Reply:
Then ... what for empty string?

Reply:
it seems that empty string has no reason to be here

removed it

Reply:
I thought maybe it would be better to remove null from config and use empty string (since empty string is the value in quick picker associated with go to file) - map it to null in code.